miner/bidder.go

```diff
@@ -111,28 +111,36 @@ func (b *Bidder) mainLoop() {
 	defer timer.Stop()
 	<-timer.C // discard the initial tick
 
-	currentHeight := b.chain.CurrentBlock().Number.Int64()
+	var (
+		bidNum        int64 = 0
+		bidUntil      time.Time
```

Comment:
I will suggest that we split the time range [now, bidUntil] into three pieces, when each piece reach, it will tick and choose a best bid so far and send

Reply:
The current logic is performing the same tasks